### ISSUE TITLE
github release function

### ISSUE THREAD

**Issue by shibumi** [SirVer/ultisnips#833]
Hello,
I wonder why there haven't been a release since december 2015? Is there any reason for this?
It would be nice if you could continue the old behaviour of the old maintainer and use the release function.
Many distributions get their releases via this function.

Thanks in advance

**Comment by seletskiy** [comment]
@shibumi: If it wasn't the problem since 2015, why it's now? Do you have personal experience?

**Comment by shibumi** [comment]
@seletskiy The problems are missing features in stable releases on many distributions because they all still use the 2015er version.

**Comment by anthraxx** [comment]
oh cool, it would be awesome to see a new release and updated package. but good stuff that nothing is broken so far, still looking forward to your improvements :smile:

**Comment by shibumi** [comment]
@seletskiy Any update on this?

**Comment by roxma** [comment]
Just curious, what distro is using the stable releases for vim plugins. I always use a custom `$HOME/.vimrc` and a plugin manager for  installing/updating plugins.

IMHO a release is useful sometimes, for normal users to choose a more stable one that just work. Although I've never used this with vim plugins.

**Comment by shibumi** [comment]
@roxma Arch Linux packages many vim-plugins. And I think debian/ubuntu and CentOS/RHEL does the same for 'stable releases' of vim plugins. Note that not everybody is fine with using github HEAD on the master branch.

**Comment by andreyorst** [comment]
I'm not emacs user,  but I know, that they have [melpa stable](https://stable.melpa.org/#/) (Milkypostman’s Emacs Lisp Package Archive). It seems good practice to have a _stable version_ in some source as they do, and unstable git master version in separate repository, like [melpa](https://melpa.org/#/) itself, wich by the way comes as default repo for emacs users.

So I agree that having actual stable version of ultisnips may be good, however, since there is no stable archive for vim plugins, I think that most of users who use package managers are not mind to use git version, because you're always able to rollback to any commit, if something happened, or not update plugins at all. No one forces you to do it anyway. Installing vim plugins systemwide seems weird to me.

**Comment by SirVer** [comment]
I did the last release of UltiSnips to the repository of Vim plugins: https://www.vim.org/scripts/script.php?script_id=2715

I stated there that I would not do further releases, because master of this repo is always stable. I also think it is common now that most Vim'ers use a plugin manager and track repos directly. 

However, I now think regular releases of a project gives a sense of progress. Also the release notes are useful to track major changes over time. I am all for releasing a new version soon, but I am running out of time for UltiSnips again. so no promises.
